updated the LIST identity/entity-alias/id endpoint docs (#23016)
hashicorp/vault: website/content/api-docs/secret/identity/entity-alias.mdx

## List entity aliases by ID

> This endpoint returns a list of available entity aliases and key data by their identifiers.

Reviewer: ```suggestion
The list by ID endpoint returns the available entity aliases and key data by their identifiers.
```
Style correction: avoid "this" as a pronoun

```json
{

> "data": {

Reviewer 2: Inconsistent indentation:
```suggestion
    "data": {
```
